src/components/tl/mlx5/tl_mlx5.h

```diff
@@ -164,6 +164,7 @@ typedef struct ucc_tl_mlx5_rcache_region {
     ucc_tl_mlx5_reg_t   reg;
 } ucc_tl_mlx5_rcache_region_t;
 
+//TODO: add UCC_COLL_TYPE_ALLGATHER once mcast design is completed
```

Comment:
nit: is this comment needed?

Reply:
This is just to remind ourselves. But it is not needed. Let me remove it.